pkg/audit/authreq.go

```diff
@@ -34,6 +35,14 @@ func AuthenticatorRequestWithAuditDecision(authReq authenticator.Request) authen
 			AddDecisionAnnotation(req, AllowDecision)
 		}
 
+		if res != nil && res.User != nil {
+			// This is only best effort. In the case of !ok or err, it won't have
+			// values in most cases.
+			AddUsernameAnnotation(req, res.User.GetName())
+		} else {
+			klog.V(5).Infof("could not audit user name on authentication attempt")
```

Comment:
what is the default verbosity level? we don't specify one as per https://github.com/openshift/cluster-authentication-operator/blob/master/bindata/oauth-apiserver/deploy.yaml, I believe it is 4.

what is the value of having this in logs? if it is unset in the deployment most likely this will never ever be read out. i believe introducing a metric is more sensible in this case. we don't spam logs and can correlate requests without user names.

Reply:
I'll remove it.